curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/666aae6a5d30a71f1fd7749f.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 666aae6a5d30a71f1fd7749f
+title: Step 6
+challengeType: 20
+dashedName: step-6
+---
+
+# --description--
+
+The `insert` method can add an element at any position in a list, the first argument is the position at which the element has to be added, and the second argument is the element to add. For example, let's add a new element in the third position of `example_list`.
```

Comment:
```suggestion
The `.insert()` method can add an element at any position in a list, the first argument is the position at which the element has to be added, and the second argument is the element to add. For example, let's add a new element in the third position of `example_list`.
```
Considering what we said about making consistent the way we refer to methods/functions. Not sure about the dot, at this point.